updated checkout proc for release milestone 5

chrislomonico opened the pull request:
## What this PR changes/adds

Small doc change to indicate the correct version of the `edc` to use when building the mvd

## Why it does that

Documentation was out of sync with the current process


## Linked Issue(s)

Closes #273

chrislomonico committed:
updated checkout proc for release milestone 5

pkirch commented on system-tests/README.md:
````diff
@@ -57,8 +57,8 @@ System tests are run both in local deployment (using docker compose) and in the
 ```bash
-# EDC (in the EDC root folder)
-git checkout 3ff940b720f44826df28e893fb31344eb6faacef
+# EDC (in the EDC root folder, checkout the release tag to a local branch)
````
Why creating a new branch locally? Why only for EDC and not the other repos?

chrislomonico replied:
because when you checkout a `tag` one must specify a branch name

pkirch commented on system-tests/README.md:
```diff
@@ -57,8 +57,8 @@ System tests are run both in local deployment (using docker compose) and in the
-# EDC (in the EDC root folder)
-git checkout 3ff940b720f44826df28e893fb31344eb6faacef
+# EDC (in the EDC root folder, checkout the release tag to a local branch)
+git checkout tags/v0.0.1-milestone-5 -b ms5
```
Should that be `v0.0.1-milestone-5.1` as in line 36?

```suggestion
git checkout tags/v0.0.1-milestone-5.1 -b ms5
```

pkirch reviewed (approved):
Minor comments.

chrislomonico committed:
Update system-tests/README.md

I will update this however it was stated that we would be pinned to version 5; what needs clarity is why we are now on 5.1 and are we including minor rev's for the milestones as part of the same version (for example 5 vs 5.1)

Co-authored-by: Peter Kirchner <pkirch@users.noreply.github.com>
